api/api_helpers.py

```diff
@@ -0,0 +1,32 @@
+"""File holding some reusable methods used in the API addon submission tests"""
+
+
+def verify_addon_response_details(payload, response, request):
+    """Method checking that the values set in the request payload are found
+    in the response returned by the API. It can be used both for verifying
+    new uploads and addon edits, based on the <request> argument.
+    Note that payload keys and the response keys structure is different in some places."""
+    assert payload['categories'] == response['categories']
+    assert payload['name'] == response['name']
+    assert payload['slug'] == response['slug']
+    assert payload['description'] == response['description']
+    assert payload['summary'] == response['summary']
+    assert payload['developer_comments'] == response['developer_comments']
+    assert payload['homepage'] == response['homepage']['url']
+    assert payload['support_email'] == response['support_email']
+    assert payload['is_experimental'] == response['is_experimental']
+    assert payload['requires_payment'] == response['requires_payment']
+    assert payload['tags'] == response['tags']
```

Comment:
Why not do a list comparison?

Reply:
@jrbenny35 I've changed this to a list comparison but the process is not as straight forward as you might have imagined. The process involves extracting values from dictionaries/ JSON in order to create list of items that we want to compare. 
This is the result - https://github.com/mozilla/addons-release-tests/pull/560/files#diff-e6ddbb4de1af2b5fa9d96a0471cdee84046ebec610db094e528772d18fbe2e8f